iac/example.yml

```diff
@@ -23,8 +23,7 @@ jobs:
           # or you can signup for free at https://snyk.io/login
           SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
         with:
-          # Scan the whole repository
-          file: . 
```

Comment:
I think we should add back this line because without it `with:` on L25 doesn't have any arguments.
I do agree that the changes you made in L26 are good and should be kept.

Reply:
Good catch, the `with` key should have been removed as it didn't have any keys now.  
The removal of the `file` key was intentional to highlight the default use in which no path is specified and the whole repository is scanned.

Reply:
If you are removing the `file` the comments in this area don't make a lot of sense